pkg/deploy-manager/config.go

```diff
@@ -93,6 +104,9 @@ func NewDeployManager() (*DeployManager, error) {
 		return nil, err
 	}
 
+	// controller-runtime client
+	crClient, err := crclient.New(config, crclient.Options{Scheme: scheme.Scheme})
```

Comment:
good usage of the crClient here. I wish I had known about this client earlier. Eventually we'll transition all the functest logic to us it.